src/main/java/school/lemon/changerequest/java/Employee.java

```diff
@@ -0,0 +1,50 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 15.12.16.
+ */
+public class Employee {
+    private String name;
+    private double salary;
+    private int workedHours;
+    public static final int hoursInMonth = 160;
+
+    public Employee() {
+    }
+
+    public Employee(String name) {
+        this.name = name;
+    }
+
+    public void setWorkedHours(int workedHours) {
```

Comment:
From my point of view, any Employee can do some amount of work many times and with your implementation you can only set some amount of hours employee worked already. I would prefer some kind of `work(int hours)` method, that would add some amount of `hours` to already worked this month hours.